routing/router.go

```diff
@@ -1261,7 +1261,7 @@ func (r *ChannelRouter) processUpdate(msg interface{},
 				"existence: %v", err)
 		}
 		if isZombie {
-			return newErrf(ErrIgnored, "ignoring msg for zombie "+
+			return newErrf(ErrZombie, "ignoring msg for zombie "+
```

Comment:
I don't _think_ this should cause any issues...

Can remove it if wanted though, just felt like we should start to distinguish outdated channel updates from zombies. I think it's the case though that a zombie shouldn't even get this far down as the gossiper filters it out at an earlier step.

Reply:
Nothing stops an implementation from just sending arbitrary gossip to us, so we should look for any places that depend on `ErrIgnored` just to be safe (gossiper and router come to mind) and see if `ErrZombie` applies there as well.

Reply:
I think I'll just remove it to isolate the set of changes in this diff.